audio_acq_node:
  ros__parameters:
    n_channels: 7
    src: 'hw:5,0'
    sample_rate: 48000
    hop_size: 4800
    frame_size: 48000
    microphone_frame_id: 'kinect_frame'

audio_proc_node:
  ros__parameters:
    n_channels: 7
    src: 'hw:5,0'
    sample_rate: 48000
    hop_size: 4800
    frame_size: 48000
    microphone_frame_id: 'kinect_frame'